OpenRA.Mods.Common/Traits/Render/WithResupplyAnimation.cs

```diff
@@ -0,0 +1,92 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2017 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits.Render
+{
+	[Desc("Replaces the building animation when it repairs a unit.")]
```

Comment:
repairs/rearms or resupplies.

Reply:
Well spotted. Guess I'll replace "building animation" with "default animation" too, since one of the goals is to make this work with non-buildings as well.